05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,3 +1,50 @@
+.box {
+  float: left;
+  width: 500px;
+}
+
+.button {
+font-weight: bold;
+color: #252b3b;
+border: 2px solid #555;
+padding: 5px;
+text-decoration: none;
+display: inline-block;
+}
+
+a {
+  color: #555;
+  text-align: center;
+  text-decoration: none;
+  margin: auto;  
+}
+
+.who {
+  background-color: rgb(240,240,240);
+  text-align: center;
+  width: 500px;
+  float:right;
```

Comment:
if you have two boxes next to each other, they should be floating in the same direction, or else they might not be aligned.